hack/conformance-test.sh

```diff
@@ -98,13 +96,19 @@ echo "Conformance test checking conformance with Kubernetes version 1.0"
 #  Restart: node management.
 #  Scale: performance
 #  Services.*load balancer: not all cloud providers have a load balancer.
-#  Services.*NodePort: flaky
+#  Services.*NodePort: requires you to open the firewall yourself, so not covered.
+#  Services.*nodeport: requires you to open the firewall yourself, so not covered.
```

Comment:
This test also required an external load balancer, see https://github.com/GoogleCloudPlatform/kubernetes/pull/10999